core/modules/proto/worker.proto

```diff
@@ -31,9 +31,9 @@ message TaskMsg {
     // Future: might have multiple db/chunk dependencies.
     optional string db = 2;
     optional int32 chunkid = 3;
-    repeated string scantables = 4;
```

Comment:
You might want to leave the deprecated field in the source but commented out with a note, so people know what was originally in that slot if they ever see one in the wild, and to discourage accidental re-use?